### ISSUE TITLE
I used the plugin to remove sections from my code using a regexp, is this still possible somehow?

### ISSUE THREAD

**Issue by enex** [artembatura/modify-source-webpack-plugin#85]
```javascript
         new ModifySourcePlugin({
            rules: [
               {
                  test: /\.tsx?$/,
                  modify: (src) =>
                     src.replace(/(?<=\/\/\s*#region ssr)[\s\S]*?.*?(?=\/\/\s*#endregion)/g, ""),
               },
            ],
         })
```

**Comment by artembatura** [comment]
Hi! In last stable version (v4) unfortunately not. But recently I was working on [new version](https://github.com/artembatura/modify-source-webpack-plugin/pull/83) (v5) that should have functionality for replacing text ranges and line ranges. And released it under `5.0.0-beta.1` version. There still is no official docs, but I can help you try it out right here.

In your example it will be:

```js
new ModifySourcePlugin({
  rules: [
    {
      test: /\.tsx?$/,
      operations: [
        new ReplaceOperation(new MarkedLinesRange("// #region ssr", "// #endregion"), "", "all")
      ]
    }
  ]
})
```

Note that it's not final API and can be changed later.

**Comment by enex** [comment]
Thank you, that works 👍🏻
